docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,190 @@
+==============================
+Tutorial: Moving a Data Volume
+==============================
+
+.. note:: This tutorial takes roughly 30 minutes, but because there are a few things to download, times might vary depending on the speed of your connection.
+
+.. contents:: 
+   :local:
+   :backlinks: none
+   :depth: 2
+
+You will use Flocker to migrate a Docker container with its data volume from one host to another.
+The container you move will be part of a two-container application, the other container will not move and the two will remain connected even when they are on different hosts.
+
+To begin the tutorial you will first install the Flocker client on your local machine, then install Flocker onto two hosts.
+You will then be ready to use Flocker to migrate a Docker container with a volume attached from one host to the other.
+
+You will be controlling your Flocker cluster via the CLI you've installed locally.
```

Comment:
"you've installed locally" - I haven't. Maybe this counts as future perfect ("you will have installed locally" ) because of the will at the beginning of the sentence but it isn't clear.